CppExport/src/dependency_analysis/DependencyUnit.h

```diff
@@ -46,6 +48,11 @@ class CPPEXPORT_API DependencyUnit
 	private:
 		QString name_;
 		Model::Node* node_;
+		QList<DependencyTarget> targets_;
```

Comment:
The thing is that this does not yet refer to dependency units. It is just the targets of references occuring in the current dependency unit. The dependency on other units is supposed to be computed after the merge step as far as I understood.